kibana/Tiltfile

```diff
@@ -0,0 +1,27 @@
+load('ext://helm_remote', 'helm_remote')
+load('ext://namespace', 'namespace_create')
+
+namespace_create('elastic-system')
+
+operator_yaml = local('curl -sSL https://download.elastic.co/downloads/eck/1.1.2/all-in-one.yaml')
```

Comment:
Any reason not to include this YAML in the repo and remove a network dependency?

Reply:
mostly it's 100K of yaml and isn't pedagogically that interesting

(though it does slow down the facets UI :grimacing: )